### ISSUE TITLE
unable to spawn executable when dumping app extensions

### ISSUE THREAD

**Issue by asdfzxcvbn** [ChiChou/bagbak#105]
**Describe the bug**
when dumping snapchat, the bundle will be dumped correctly. however, this will happen when dumping the app extensions:

```
[decrypt] Snapchat
[decrypt] PlugIns/SnapchatHomeScreenWidget.appex/SnapchatHomeScreenWidget
[decrypt] PlugIns/SnapchatIntentsExtension.appex/SnapchatIntentsExtension
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: Unable to spawn executable at '/private/var/containers/Bundle/Application/AAD2D010-58DC-4E8C-9057-865D30392D5D/Snapchat.app/PlugIns/SnapchatLocationPushExtension.appex/SnapchatLocationPushExtension': Undefined error: 0]

Node.js v18.16.0
```

**To Reproduce**

1. `bagbak Snapchat`

**Expected behavior**
the app is dumped successfully.

**Desktop (please complete the following information):**
 - OS: macos mojave
 - nodejs: v18.16.0
 - frida-node: v16.0.19
 - frida on device version: v16.0.19
 - iOS and jailbreak version: iOS 14.8, palera1n
 - The app you are trying to work on: Snapchat

**Additional context**
this is the only app i've seen this happen on.

**Comment by ChiChou** [comment]
@asdfzxcvbn Snapchat has a malformed package. 

Workaround: SSH to device, `chmod +Xx /private/var/containers/Bundle/Application/AAD2D010-58DC-4E8C-9057-865D30392D5D/Snapchat.app/PlugIns/SnapchatLocationPushExtension.appex/SnapchatLocationPushExtension`

**Comment by ChiChou** [comment]
Fixed in v3.0.3

https://github.com/ChiChou/bagbak/commit/ceca864163c9f0c40ce2fdc21690b08a8e87f052
